### PR TITLE
ClusterLoader - Adding mapping overrides

### PR DESCRIPTION
**Pull request by krzysied**
ref #281

### CODE REVIEW

File: clusterloader2/pkg/config/codec.go [hunk 1]
```diff
@@ -44,3 +44,10 @@ func convertToObject(raw []byte) (*unstructured.Unstructured, error) {
 	}
 	return obj, nil
 }
+
+func convertToGivenStruct(raw []byte, v interface{}) error {
```
Comment (wojtek-t) [inline clusterloader2/pkg/config/codec.go]: decodeInto

Comment (krzysied) [reply clusterloader2/pkg/config/codec.go]: Done.

**Pull request merged**